.github/workflows/code_scan.yml

```diff
@@ -0,0 +1,21 @@
+name: Semgrep CI
```

Comment:
Minor (ignorable): Is there a more general name for this workflow that might help people understand what it is (and what has failed when it raises errors)? e.g. `Code Scanning CI`?

Reply:
```suggestion
name: Code Scanning
```